src/python/pants/backend/scala/compile/scalac.py

```diff
@@ -98,7 +104,7 @@ async def compile_scala_source(
         component_members_with_sources,
         await MultiGet(
             Get(
-                SourceFiles,
+                StrippedSourceFiles,
```

Comment:
For SemanticDB plugin to work properly, we need to use stripped source files, otherwise the emitted `.semanticdb` output files will not have the correct path.

Reply:
This would be useful as a comment in the code.